### ISSUE TITLE
Freya script 

### ISSUE THREAD

**Issue by mcgbg** [ih1115/BattleriteAimbot#3]
Thank you so much for the script. Has Freya released a script?

**Comment by ih1115** [comment]
For champions like Freya I believe the aim-bot alone should be enough (you just press buttons and it aims at the closest enemy).

I'm not sure though since I am new to the game (I don't know Freya's abilities).

I guess I could write something for Freya that automatically throws the hammers or jumps to target if they're in range.

The program as it is doesn't detect if enemy is shooting, countering, etc. so it would be useless to "auto counter" at the moment.

Either way, I don't play Freya so it isn't a priority of my own but if someone else does do something for Freya feel free to make a pull request.

TL;DR 
No Freya script until the program can detect players shooting / countering / stunned.
